src/Mono.Android/Mono.Android.csproj

```diff
@@ -319,6 +319,7 @@
     <Compile Include="Xamarin.Android.Net\AuthModuleDigest.cs" />
     <Compile Include="Xamarin.Android.Net\IAndroidAuthenticationModule.cs" />
     <Compile Include="Xamarin.Android.Net\OldAndroidSSLSocketFactory.cs" />
+    <Compile Remove="Test\**" />
```

Comment:
When I opened this in VS:

![image](https://user-images.githubusercontent.com/840039/77167953-fd880a80-6a84-11ea-9ba0-f6f0e2cc0462.png)

So I think we also need:
```xml
<None Remove="Test\**" />
<EmbeddedResource Remove="Test\**" />
```

Reply:
Added